lib/templates/plugin.js

```diff
@@ -132,5 +132,8 @@ export default (ctx, inject) => {
         }
         return jsCookie.get(tokenName)
     }
-  })
+  }
+
+  ctx.$apolloHelpers = apolloHelpers
```

Comment:
It makes inject, is not it?

Reply:
Not bound to ctx so the ide is not recognized.